tools/generators/recipe-generator/index.ts

```diff
@@ -0,0 +1,49 @@
+import * as path from 'path';
+import { Tree, formatFiles, generateFiles, readWorkspaceConfiguration } from '@nrwl/devkit';
+import { RecipeGeneratorGeneratorSchema } from './schema';
+import { getProjectConfig } from '../../utils';
+
+export default async function (tree: Tree, schema: RecipeGeneratorGeneratorSchema) {
+  const validatedSchema = validateSchema(tree, schema);
+
+  const { recipesRoot, ...normalizedOptions } = normalizeOptions(tree, validatedSchema);
+
+  generateFiles(tree, path.join(__dirname, 'files'), recipesRoot, normalizedOptions);
```

Comment:
nit: use `joinPathFragments` method from `@nrwl/devkit` instead
```suggestion
  generateFiles(tree, joinPathFragments(__dirname, 'files'), recipesRoot, normalizedOptions);
```